src/strategies/FluentStrategyMixins.ts

```diff
@@ -49,7 +49,8 @@ export function Cached<TBase extends MixinStrategy>(Base: TBase) {
 export function Biased<TBase extends MixinStrategy>(Base: TBase) {
   return class extends Base {
     buildArbitraryCollection<A>(arbitrary: Arbitrary<A>, sampleSize = this.configuration.sampleSize): FluentPick<A>[] {
-      return this.isDedupable() ? arbitrary.sampleUniqueWithBias(sampleSize) : arbitrary.sampleWithBias(sampleSize)
+      return this.isDedupable() ? arbitrary.sampleUniqueWithBias(sampleSize, this.prng.generator) :
```

Comment:
Maybe a newline after "?"

Reply:
The second line is still too long according to lint standards, that's why I did it like this, I can however turn it into 3 different lines one for the check, one for the true case, and another for the false case.